### ISSUE TITLE
Printable area shifts up by one upon cloning the worksheet

### ISSUE THREAD

**Issue by Ataraxia-Mechanica** [troldal/OpenXLSX#217]
![image](https://github.com/troldal/OpenXLSX/assets/111327195/8aee6dfc-ccc9-4b8e-94e5-61a866b91c1a)

Original worksheet:
![image](https://github.com/troldal/OpenXLSX/assets/111327195/e5ee4673-a6a4-4242-a5a9-1f711d927085)

After cloning:
![image](https://github.com/troldal/OpenXLSX/assets/111327195/2ce1dcd7-a38d-4ebb-ab2d-6814ac1566bd)

**Comment by Ataraxia-Mechanica** [comment]
Versions: 
Office 2021
OpenXLSX 0.3.2
C++23 (Clang 16.0.2)

**Comment by Ataraxia-Mechanica** [comment]
Well, more like shifting to the top, except when the area includes the first row, then the last row is trimmed instead

Before:
![image](https://github.com/troldal/OpenXLSX/assets/111327195/c55594a5-8796-4ffc-981c-d0afd278ca69)

After:
![image](https://github.com/troldal/OpenXLSX/assets/111327195/1af1bc83-5dff-4861-b835-e7a40975bdd3)

**Comment by Ataraxia-Mechanica** [comment]
...Except when it doesn't want to do that for whatever reason.
Before:
![image](https://github.com/troldal/OpenXLSX/assets/111327195/fff76378-fa2e-43bc-a27b-878413143262)
After:
![image](https://github.com/troldal/OpenXLSX/assets/111327195/f051d267-ea2e-4c24-b938-afc7bc29c38c)

![table-hit-yakuza](https://github.com/troldal/OpenXLSX/assets/111327195/a7e21d2a-4b7c-4960-b894-4a1b1ab5cf70)

**Comment by Ataraxia-Mechanica** [comment]
Looks like it has something to do with empty cells. when the last row has some text in it it no longer shifts up.

**Comment by aral-matrix** [comment]
Hi @Ataraxia-Mechanica - I am going through the backlog of issues at the moment, and I could not reproduce your issue - when I use cloneSheet, the resulting sheets are identical - but the print ranges are not cloned, they remain defined for the original worksheets - but for the new worksheets no print ranges are copied / defined.

But I guess you were referring to the "view" -> "page breaks" (what it's called in LibreOffice) - and at least in LibreOffice, the print area immediately changes (while editing) when the last value from an otherwise empty row/column is removed.

It appears what you were seeing was unrelated to cloning a sheet. I wonder if MS Office possibly behaves different here, but either way I can assure you the underlying XML after a sheet cloning operation is identical - here is an example diff applied to Demo10:
```
$ diff -r xl/worksheets/ ../10.1/xl/worksheets/
Only in xl/worksheets/: sheet2.xml
```
where sheet2 is the cloned worksheet.

Closing this as "superseded", so as not to label it invalid. I would kindly ask you to open a new issue with a problem description should you still observe something "going wrong" during cloning.

I apologize for the long wait until you got a reply.

**Comment by Ataraxia-Mechanica** [comment]
Yeah it has to be some jank on the Office side if the XML is identical. Couldn't remember what happened for something this long ago and I eventually found a workaround, so no worries for the long delay.

**Comment by aral-matrix** [comment]
Thank you for the close-out feedback :)
